packages/styles/scss/components/leadspace/_leadspace.scss

```diff
@@ -492,6 +501,32 @@ $btn-min-width: 26;
       height: rem(640px);
     }
   }
+
+  :host(#{$dds-prefix}-leadspace-image),
+  .#{$prefix}--image {
```

Comment:
@NSDrowned this is probably what's causing the other images to mess up, this is not specific enough

Reply:
That was it, thanks @jeffchew !